src/envoy/transcoding/filter.cc

```diff
@@ -184,6 +193,29 @@ Http::FilterTrailersStatus Instance::encodeTrailers(Http::HeaderMap& trailers) {
     if (data.length()) {
       encoder_callbacks_->addEncodedData(data);
     }
+
+    if (!method_->server_streaming()) {
+      const Http::HeaderEntry* grpc_status_header = trailers.GrpcStatus();
+      if (grpc_status_header) {
+        uint64_t grpc_status_code;
+        if (!StringUtil::atoul(grpc_status_header->value().c_str(),
```

Comment:
what is this line for?   just check status_code is integer?   after atoul, grpc_status_code is not used.